src/app/components/info-card/info-card.tsx

```diff
@@ -68,35 +69,38 @@ export function InfoCardAssetValue({
   fiatSymbol,
   symbol,
   icon,
+  ...props
 }: InfoCardAssetValueProps) {
   return (
-    <Stack
-      mb="44px"
-      width="100%"
-      alignItems="center"
-      backgroundColor="#F9F9FA"
-      py="24px"
-      border="1px solid #EFEFF2"
-      borderRadius="10px"
-    >
-      {icon && <Box as={icon} size="32px" />}
+    <Box width="100%" px="loose">
```

Comment:
Just a note that, if we ever want this component without the `px` it becomes as hassle. Better to add it separately on each use?

Reply:
yeah, agree here, done